chart/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: aws-event-sources
-version: 0.1.0
-appVersion: 0.2.0
+version: 0.2.0
```

Comment:
Considering the only change in the  chart is a fixed typo, maybe 0.1.1 is more aligned with the definition of semver?

Reply:
The controller and adapter image versions is updated in this chart version from `v0.2.0` to `v0.3.0`  (it's not a typo)

Reply:
The component are 0.3.0, but the chart is versioned individually from the appVersion.

Reply:
okay. that makes sense. i'll bump the patch version instead